crates/autopilot/src/infra/solvers/mod.rs

```diff
@@ -172,3 +177,29 @@ pub async fn response_body_with_size_limit(
     }
     Ok(bytes)
 }
+
+/// Try to notify all the non-settling solvers in a background task.
+pub fn notify_non_settling_solvers(
```

Comment:
Seems like it creates a bunch of complexity to bundle these updates together. Why not just fire and forget the request to each driver as they get deny listed?